bookwyrm/migrations/0154_bookauthor.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 3.2.14 on 2022-07-09 21:09
+
+import bookwyrm.models.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("bookwyrm", "0153_merge_20220706_2141"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="BookAuthor",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created_date", models.DateTimeField(auto_now_add=True)),
+                ("updated_date", models.DateTimeField(auto_now=True)),
+                (
+                    "remote_id",
+                    bookwyrm.models.fields.RemoteIdField(
+                        max_length=255,
+                        null=True,
+                        validators=[bookwyrm.models.fields.validate_remote_id],
+                    ),
+                ),
+                (
+                    "author_type",
+                    models.CharField(
```

Comment:
Would love to have "Narrator" for Audiobooks. I know, there is no schema.org property for this - which is a problem from schema.org as they never went with time. 

For Graphic Novels it would be great if there would be "Lettering" as well and in general: "Cover Artwork" as this does not need to be a by the person which did the illustrations.

Reply:
Thank you! Agreed

Reply:
And if we really want to show appreciation to authors: Author of Preface (which has no schema.org representation, only in ONYX)

Reply:
I stumbled about the lack of an editor (in scientific books there often is one ore more editors who compiled the book by adding pieces from different authors). At the moment the first author is often the only one mentioned, but it should be the editor.